### DISCUSSION TITLE
How to remove categories?

### DISCUSSION THREAD

**Discussion by Play-Tiger** [searxng/searxng#1467] [Q&A] [answered]
I tried to remove categories, but I failed to do so. I copied the example in the documentation and then removed every categorie I don't need. 
![image](https://user-images.githubusercontent.com/76755673/177585100-bb21d1f4-d630-4da5-be3a-d71f24da02ad.png)
That didn't work.
My next try was to write a false behind every categorie i don't need. That also didn't work.
Can somebody explain to me how I can remove certain categories?

**Comment by return42** [comment]
I assume you are using `use_default_settings` .. this seems an issue to me, could you please open a [bug report](https://github.com/searxng/searxng/issues/new?assignees=&labels=bug&template=bug-report.md&title=) .. thanks!

**Reply by dalf** [reply]
Somewhat related to https://github.com/searxng/searxng/issues/1020 : 
The "usual" way is to remove engine, and then only categories with engines are shown. But as reported in the issue 1020, this does not work.

**Reply by return42** [reply]
> The "usual" way is to remove engine, and then only categories with engines are shown.

not sure I understand what you mean, PR #634 says:

*Previously all categories were displayed as tabs in the UI. This PR changes this so that only categories listed under `categories_as_tabs` in `settings.yml` are displayed.*

**Reply by Play-Tiger** [reply]
> I assume you are using `use_default_settings` .. this seems an issue to me, could you please open a [bug report](https://github.com/searxng/searxng/issues/new?assignees=&labels=bug&template=bug-report.md&title=) .. thanks!

Yes, I do use `use_default_settings`.
This is my settings.yml file: 
![image](https://user-images.githubusercontent.com/76755673/177635002-282e72b7-768e-497d-b54b-62a1df91ca8f.png)

**Reply by Play-Tiger** [reply]
So you think that's not a issue on my end, but a bug?

**Reply by return42** [reply]
I think it is a bug but @dalf says its not a bug .. but I didn't understand what he means.  In https://github.com/searxng/searxng/discussions/1467#discussioncomment-3094883 I asked @dalf to clarify.

Lets wait what @dalf will respond to my question.

**Comment by return42** [comment, marked as answer]
FYI: there is an issue entry about

- https://github.com/searxng/searxng/issues/1020

and a bugfix in 

- https://github.com/searxng/searxng/pull/1545
